src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_NUMBER_OF_FIGURES = 5;
+    public static final double MAX_SIZE = 100d;
+    public static final double DEFAULT_RADIUS_OF_CIRCLE = 10d;
+    public static final String DEFAULT_COLOR = "white";
+
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public FigureSupplier(Random random, ColorSupplier colorSupplier) {
+        this.random = random;
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Figure getRandomFigure() {
+

```

Comment:
Let's remove redundant empty line.

Reply:
Then maven throws an error

Reply:
Strangely. Reloaded the project and the error is no longer there.